Makefile

```diff
@@ -37,12 +37,22 @@ lint-check:
 	flake8 --config=package/.flake8 package
 	mypy --config-file=package/mypy.ini package
 
+schema-fix:
```

Comment:
curious: what's the need for this?

Reply:
This is what automatically generates the `graphql.schema` file and png from the strawberry schema. It's called `schema-fix` (bit of a weird name I know) by analogy with `make format-fix`, which will make the `make format-check` CI pass. Here `schema-fix` will make the `make schema-check` CI that tests if the schema file and diagram is up to date pass.